state/facade.go

```diff
@@ -27,7 +27,7 @@ type Facade interface {
 	UpdateLastCertificate(v *vote.Vote) error
 	ProposeBlock(valKey *bls.ValidatorKey, rewardAddr crypto.Address, round int16) (*block.Block, error)
 	ValidateBlock(block *block.Block) error
-	CommitBlock(height uint32, block *block.Block, cert *certificate.Certificate) error
+	CommitBlock(block *block.Block, cert *certificate.Certificate) error
```

Comment:
@themantre 
it would be better to use `blk` instead of `block` based on linter warning.